lib/this_data.rb

```diff
@@ -74,7 +74,8 @@ def error(message, prefix: true)
       # Returns an HTTPResponse
       def track_with_response(event)
         response = Client.new.track(event)
-        if response.try(:success?)
+        success = response.success? rescue nil # HTTParty doesn't like `.try`
```

Comment:
Rescuing everything here could obscure other implementation issues, so instead you could do something like this:

```ruby
success = response && response.success?
```